src/Frontend/Views/Portal/Account.tsx

```diff
@@ -49,7 +61,9 @@ function AccountModal({ setOpen }: { setOpen: (open: boolean) => void }) {
           >
             <Icon type={IconType.X} />
           </button>
-          <div style={{ fontSize: '2em' }}>{twitter || truncatedAddress}</div>
+          <div style={{ fontSize: '2em' }}>
+            {twitter || <TextPreview text={address} unFocusedWidth={'50%'} focusedWidth={'100%'} />}
```

Comment:
```suggestion
            {twitter ?? <TextPreview text={address} unFocusedWidth={'50%'} focusedWidth={'100%'} />}
```

Very minor nit:

It's better practice to use the nullish coalescing operator (`??`) here because it checks for null/undefined values in the first argument before falling back to TextPreview. Whereas simply using the double pipes (OR) would fallback to TextPreview if the value is simply falsy (like 0, or an empty string). In this case, it won't really matter, but it's nice to have just in case.

Reply:
isnt null/undefined also falsy tho

Reply:
Oops, sorry I wrote the wrong thing. `||` will fallback if the first value is not truthy. So yes, null/undefined is falsy but using `||` will also fallback if the value is 0 or "", which does not always mean null/undefined.

It doesn't matter in this case. You can leave it unchanged and nothing bad will happen. It's just nice to be explicitly checking only for null/undefined.